settings.gradle.kts

```diff
@@ -2,6 +2,7 @@ rootProject.name = "ec-config-template"
 
 pluginManagement {
     repositories {
+        mavenLocal()
```

Comment:
I think this is unintentional change. We don't want customers to use `mavenLocal`